src/pages/docs/creators/ray/conversion-to-ray-on-golem-tutorial.md

```diff
@@ -181,17 +181,17 @@ pip3 install -U ray-on-golem
 Now, run the following to start a basic cluster, based on our example configuration. Your cluster will run on our testnet - it is free, but not very powerful.
 
 ```bash
-# Download the golem-cluster.yaml
-wget https://github.com/golemfactory/ray-on-golem/raw/main/golem-cluster.yaml
+# Download the dds-with-ray.yaml
```

Comment:
it was because it uses endplay lib.
it was in the yaml mostly because there was a problem with pip and this Pillow with capital P - I think
other than that it could have been in ray.init(runtime_env)
now I think that using pip in yaml is better as it doesn't need to be repeated with every submit.
so this is why we are using separate yaml :)